storage/client_raft_log_queue_test.go

```diff
@@ -51,12 +51,12 @@ func TestRaftLogQueue(t *testing.T) {
 	}
 
 	// Get the raft leader (and ensure one exists).
-	desc := mtc.stores[0].LookupReplica([]byte("a"), nil).Desc()
-	raftLeaderRepl := mtc.getRaftLeader(desc.RangeID, time.Second)
+	rangeID := mtc.stores[0].LookupReplica([]byte("a"), nil).RangeID
+	raftLeaderRepl := mtc.getRaftLeader(rangeID, time.Second)
 	if raftLeaderRepl == nil {
-		t.Fatalf("could not find raft leader replica for range %d", desc.RangeID)
+		t.Fatalf("could not find raft leader replica for range %d", rangeID)
 	}
-	originalIndex, err := raftLeaderRepl.FirstIndex()
+	originalIndex, err := raftLeaderRepl.FirstIndexUnlocked()
```

Comment:
I think I'd rather add lock/unlock pairs around all calls to `FirstIndex` et al instead of introducing `Unlocked` versions.